classes/renderables/topic_render.php

```diff
@@ -36,34 +41,39 @@ public function topic_subcription_button($lastreplyhtml = '', $currentlysubbed =
             $followingmobile = get_string('topicfollowmobile','hsuforum');
         }
 
+        if ($discussion->pinned == 1) {
+            $pinned = $OUTPUT->render_from_template('mod_hsuforum/_topic/hsuforum_topic_pinned',[]);
+        }
+
         return $OUTPUT->render_from_template('mod_hsuforum/hsuforum_topic_buttons',
             [
                 'buttonclass' => SELF::BUTTONCLASS,
-                'lastreplyhtml' => $lastreplyhtml,
+                'lastreplyhtml' => $latestpost,
                 'currentlysubbed' => $currentlysubbed,
                 'following' => $following,
-                'followingmobile' => $followingmobile
+                'followingmobile' => $followingmobile,
+                'pinned' => $pinned
             ]
         );
     }
 
     /**
      * contributors_html
-     * Takes a llist of users, and builds the avatar html based on settings.
+     * Takes a list of users, and builds the avatar html based on settings.
      * @param $users list of user avatar img tags
      * @return string
      */
-    public function contributors_html($users) {
+    public function contributors_html($discussion) {
```

Comment:
Doc block not aligning with param. Is a list of users used or a discussion fed in as a param?